src/python/grpcio_tests/tests/unit/_metadata_code_details_test.py

```diff
@@ -363,6 +380,102 @@ def testCustomCodeStreamStream(self):
         self.assertIs(_NON_OK_CODE, exception_context.exception.code())
         self.assertEqual(_DETAILS, exception_context.exception.details())
 
+    def testAbortedUnaryUnary(self):
+        self._servicer.set_code(_NON_OK_CODE)
+        self._servicer.set_details(_DETAILS)
+        self._servicer.set_call_abort()
+
+        with self.assertRaises(grpc.RpcError) as exception_context:
+            self._unary_unary.with_call(object(), metadata=_CLIENT_METADATA)
+
+        self.assertTrue(
+            test_common.metadata_transmitted(
+                _CLIENT_METADATA, self._servicer.received_client_metadata()))
+        self.assertTrue(
+            test_common.metadata_transmitted(
+                _SERVER_INITIAL_METADATA,
+                exception_context.exception.initial_metadata()))
+        self.assertTrue(
+            test_common.metadata_transmitted(
+                _SERVER_TRAILING_METADATA,
+                exception_context.exception.trailing_metadata()))
+        self.assertIs(_NON_OK_CODE, exception_context.exception.code())
+        self.assertEqual(_DETAILS, exception_context.exception.details())
+
+    def testAbortedUnaryStream(self):
+        self._servicer.set_code(_NON_OK_CODE)
+        self._servicer.set_details(_DETAILS)
+        self._servicer.set_call_abort()
+
+        call = self._unary_stream(
```

Comment:
For consistency with most of the rest of the test corpus: call this `response_iterator_call`?

Reply:
Are you suggesting changing the entire file in this PR for that? (cause this file is self-consistent and changing just the newly introduced functions will introduce additional inconsistency.) I prefer we keeping this PR as is for the time being and then cleanup for consistency across the entire code base later.

Reply:
I was suggesting changing the field name on this line, not the method name. The method name is very very consistent with the rest of the Python subcodebase.